frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vm-template-wrapper.ts

```diff
@@ -88,6 +89,8 @@ export class MutableVMTemplateWrapper extends VMTemplateWrapper {
     }
   };
 
+  hasLabel = (label: string) => _.has(this.getLabels(), label);
```

Comment:
can we use `hasIn` so we don't create new objects every time getLabels is called when the labels are missing?